apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -465,7 +468,12 @@ force_kill(ResId) ->
         undefined ->
             ok;
         Pid when is_pid(Pid) ->
+            Ref = monitor(process, Pid),
```

Comment:
we need this wait around the `remove` and `stop` calls, not just `force_kill`.
 because `force_kill` is only called in the `{error, timeout}` clause